spec/controllers/admin/moderation_controller_spec.rb

```diff
@@ -34,56 +35,123 @@ def do_patch(options = patch_options)
       end
 
       context "when moderation param is 'approve'" do
-        let(:now) { Time.current }
-        let(:deliveries) { ActionMailer::Base.deliveries }
-        let(:creator_email) { deliveries.select{ |m| m.to == %w[bazbutler@gmail.com] }.last }
-        let(:sponsor_email) { deliveries.detect{ |m| m.to == %w[laurapalmer@gmail.com] } }
-        let(:pending_email) { deliveries.detect{ |m| m.to == %w[sandyfisher@hotmail.com] } }
-        let(:duration) { Site.petition_duration.weeks }
-        let(:closing_date) { (now + duration).end_of_day }
-        let!(:sponsor) { FactoryBot.create(:sponsor, :pending, petition: petition, email: "laurapalmer@gmail.com") }
-        let!(:pending_sponsor) { FactoryBot.create(:sponsor, :pending, petition: petition, email: "sandyfisher@hotmail.com") }
+        context "and the petition is collecting signatures" do
```

Comment:
I'm failing to see what the difference is between the two contexts; they have the same test descriptions :thinking:

Reply:
I guess I was being paranoid and wanted to make sure I had covered all my bases. But yeah you're totally right.